wcfsetup/install/files/lib/system/view/user/profile/UserProfileHeaderView.class.php

```diff
@@ -146,13 +146,15 @@ private function initManagementContextMenu(): void
             new UserManagementInteractions(),
             $this->user,
             LinkHandler::getInstance()->getControllerLink(MembersListPage::class),
+            reloadHeaderEndpoint: "core/users/{$this->user->userID}/profile-header",
             configuration: new InteractionContextMenuComponentConfiguration(
                 cssClassName: 'userProfileHeader__button',
                 buttonCssClassName: 'button small',
                 dropdownMenuCssClassName: 'userProfileHeader__managementOptions',
+                tooltip: 'wcf.user.profile.management',
                 icon: FontAwesomeIcon::fromValues('gear'),
-                tooltip: 'wcf.user.profile.management'
-            )
+            ),
+            headerCssClassName: '.userProfileHeader'
```

Comment:
```suggestion
            headerCssClassName: '.userProfileHeader',
            reloadContextMenu: false
```
It is not necessary to reload the context menu, as the entire header is reloaded.